### PR TITLE
Add missing examples of ODATA URLs

### PR DESCRIPTION
**Pull request by ArvindRSingh**
.IncludeTotalCount() // is not yet supported ref. https://github.com/OData/odata.net/issues/1334

### CODE REVIEW

File: Odata-docs/client/v6/query-options.md [hunk 1]
```diff
@@ -75,38 +75,56 @@ var people = context.People.OrderBy(c => c.Trips.Count);
...
 ```
 
 ## A simple combined query combined
 
+For `GET https://host/service/EntitySet?$expand=Trips&$filter=FirstName eq 'Peter'&$orderby=Firstname asc&$skip=3&$top=3
+
 ``` csharp
 var people =
-    context.People.IncludeTotalCount()
+    context.People
```
Comment (xuzhg) [inline Odata-docs/client/v6/query-options.md]: for back-compatible, "IncludeTotalCount" is still in the codes, Right?

Comment (KenitoInc) [reply Odata-docs/client/v6/query-options.md]: @ArvindRSingh @xuzhg Currently we are supporting `IncludeTotalCount()`
In the next version, for backwards compatibility, we have kept `IncludeTotalCount()` and added `IncludeTotalCount(bool)`
We have also added V4 terms `IncludeCount()` and `IncludeCount(bool)`

Comment (KenitoInc) [reply Odata-docs/client/v6/query-options.md]: See this merged PR https://github.com/OData/odata.net/pull/1658

**Comment by xuzhg** [comment]
Merged manually. Close it.

**Pull request closed without merging**